packages/core/stdlib/math/main.browse

```diff
@@ -22,41 +40,76 @@ rule toPrecision {
 }
 
 # Math.* fns
+#*Returns the absolute value of x.
 rule abs { bind x; return (native:fn abs $x) }
+#*Returns the arccosine of x.
 rule acos { bind x; return (native:fn acos $x) }
+#*Returns the hyperbolic arccosine of x.
 rule acosh { bind x; return (native:fn acosh $x) }
+#*Returns the arcsine of x.
 rule asin { bind x; return (native:fn asin $x) }
+#*Returns the hyperbolic arcsine of a number.
 rule asinh { bind x; return (native:fn asinh $x) }
+#*Returns the arctangent of x.
 rule atan { bind x; return (native:fn atan $x) }
+#*Returns the hyperbolic arctangent of x.
 rule atanh { bind x; return (native:fn atanh $x) }
+#*Returns the arctangent of the quotient of its arguments.
 rule atan2 { bind y x; return (native:fn atan2 $y $x) }
-rule cbrt { bind x; return (native:fn cbrt $x) }
+#*Returns the cube root of x.
+rule cbrt { bind x; return (native:fn cbrt $x) }a
```

Comment:
typo (there's a random `a` at the end)

If you use the vscode extension and use the formatter, it's really easy to catch stuff like this :)

Reply:
I fixed it during a rebase